fee_allocator/accounting/settings.py

```diff
@@ -20,7 +20,7 @@ class Chains(Enum):
 CORE_POOLS_URL = "https://raw.githubusercontent.com/BalancerMaxis/multisig-ops/main/config/core_pools.json"
 
 REROUTE_CONFIG_URL = (
-    "https://raw.githubusercontent.com/BalancerMaxis/multisig-ops/main/config/core_pools_rerouting.json"
+    "https://raw.githubusercontent.com/BalancerMaxis/multisig-ops/rerouting/config/core_pools_rerouting.json"
```

Comment:
Make sure to first merge multi-sig ops rerouting branch to main again but seems to work from that branch